<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>c_membermarket</class>
 <widget class="QMainWindow" name="c_membermarket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pB_back">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>86</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;返回</string>
    </property>
   </widget>
   <widget class="QTableView" name="merchandise_view">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>511</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QStackedWidget" name="merchandiseinfo_table">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>110</y>
      <width>211</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_7"/>
    <widget class="QWidget" name="page_8">
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>商品号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="new_merchandiseNumber">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="new_stock">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>购买量</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>购买商品</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pB_create_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>210</y>
        <width>71</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>购买</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
